### PR TITLE
add master-compliance-test and master-snapshot for K8S-iac

### PR DESCRIPTION
**Pull request by shahinshirvani**

### CODE REVIEW

File: kubernetes/iac/master-compliance-test.json [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+                            "remediationDescription":"Create a PSP as described in the Kubernetes documentation, ensuring that the .spec.runAsUser.rule is set to either MustRunAsNonRoot or MustRunAs with the range of UIDs not including 0.",
+                            "remediationFunction":"PR-K8S-0015-DCL.py"
+                        }
+                    ],
+                    "title": "Do not admit root containers",
+                    "description": "Do not generally permit containers to be run as the root user.",
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": ["CIS 140"],
```
Comment (farchide) [inline kubernetes/iac/master-compliance-test.json:59]: @shahinshirvani change all `CIS 140` to `CIS`

File: kubernetes/iac/master-compliance-test.json [hunk 2]
```diff
@@ -0,0 +1,563 @@
+{
```
Comment (farchide) [inline kubernetes/iac/master-compliance-test.json:1]: @shahinshirvani file structure is not correct. there should be one master test name, one cases and all tests should be under cases

**Review by farchide — CHANGES_REQUESTED** [review]
@shahinshirvani file structure is not correct

**Comment by shahinshirvani** [comment]
i fixed these issues about change tag and structure

**Review by farchide — APPROVED** [review]

**Pull request merged**